Sources/SwiftProtobuf/JSONDecoder.swift

```diff
@@ -24,12 +25,15 @@ internal struct JSONDecoder: Decoder {
     throw JSONDecodingError.conflictingOneOf
   }
 
-  internal init(source: UnsafeBufferPointer<UInt8>) {
-    self.scanner = JSONScanner(source: source)
+  internal init(source: UnsafeBufferPointer<UInt8>, options: JSONDecodingOptions?) {
```

Comment:
Instead of making this optional, how about making it non-optional with a default value of `JSONDecodingOptions()`? (Throughout.)